### PR TITLE
TF Keras local_test dataformat compatible test case updated

### PR DESCRIPTION
**Pull request by Dayananda-V**

### CODE REVIEW

File: tensorflow/python/keras/layers/local_test.py [hunk 1]
```diff
@@ -20,216 +20,226 @@
...
-                                  keras.layers.LocallyConnected1D,
-                                  **kwargs)
-              else:
-                testing_utils.layer_test(
-                    keras.layers.LocallyConnected1D,
-                    kwargs=kwargs,
-                    input_shape=(num_samples, num_steps, input_dim))
-
-  def test_locallyconnected_1d_regularization(self):
+      for strides in [1]:
```
Comment (tomerk) [inline tensorflow/python/keras/layers/local_test.py]: Is there a point to leaving this for loop & the continue if we're cleaning up this file anyway?

Comment (Dayananda-V) [reply tensorflow/python/keras/layers/local_test.py]: @tomerk 

No, just keep the existing logic without much changes in parameterized parameter.

**Review by tomerk — APPROVED** [review]

**Comment by gbaned** [comment]
@Dayananda-V could you please fix build errors? Thanks!

**Comment by Dayananda-V** [comment]
@gbaned 

Error is not induce PR code changes, will you retrigger build one more time. TIA
`ImportError: No module named 'tensorflow_estimator.python.estimator.tpu'`

**Pull request merged**